xarray/core/common.py

```diff
@@ -341,15 +341,21 @@ def _calc_assign_results(
                 results[k] = v
         return results
 
-    def assign_coords(self, **kwargs):
+    def assign_coords(self, coords=None, **coords_kwargs):
         """Assign new coordinates to this object.
 
         Returns a new object with all the original data in addition to the new
         coordinates.
 
         Parameters
         ----------
-        kwargs : keyword, value pairs
+        coords : dict, optional
```

Comment:
Great, I don't think you need to repeat the description - you could use the same format as https://github.com/pydata/xarray/blob/master/xarray/core/dataset.py#L1864

Reply:
Thanks for the feedback. Done.